readme.md

```diff
@@ -17,14 +17,14 @@ Getting started? Play directly with the Babylon.js API using our [playground](ht
 - <https://cdn.babylonjs.com/babylon.js>
 - <https://cdn.babylonjs.com/babylon.max.js>
 
-Additional references can be found on <https://cdn.babylonjs.com/xxx> where `xxx` is the folder structure you can find in the /dist folder like <https://cdn.babylonjs.com/gui/babylon.gui.min.js>
+Additional references can be found on <https://cdn.babylonjs.com/xxx> where `xxx` is the folder structure you can find [here](https://github.com/BabylonJS/Babylon.js/tree/preview/dist) like <https://cdn.babylonjs.com/gui/babylon.gui.min.js>

```

Comment:
cc @RaananW as I guess this is not accurate anymore

Reply:
The list of available packages can be found here
https://doc.babylonjs.com/divingDeeper/developWithBjs/frameworkVers#cdn-current-versions

Reply:
Updated the PR.